openshift/templates/email-verification-service/email-verification-service-deploy.yaml

```diff
@@ -4,15 +4,61 @@ apiVersion: v1
 metadata:
   name: ${NAME}
 objects:
-
+- kind: ExternalNetwork
+  apiVersion: security.devops.gov.bc.ca/v1alpha1
+  metadata:
+    name: ${NAME}${SUFFIX}
+    network: ${NAME}${SUFFIX}
+    labels:
+      name: ${NAME}${SUFFIX}
+      network: ${NAME}${SUFFIX}
+      app: ${APP_NAME}${SUFFIX}
+      app-group: ${APP_GROUP}
+      env: ${TAG_NAME}
+  spec:
+    description: |
+      Define the network parameters for accessing smtp server
+    entries:
+      - 0.0.0.0/0
```

Comment:
Please narrow this down to the actual host name of the SMTP service, but make sure the host name does not make it into source control.
Example;
- https://github.com/bcgov/family-law-act-app/blob/master/openshift/templates/backup/backup-deploy.yaml#L22
- https://github.com/bcgov/family-law-act-app/blob/master/openshift/templates/backup/backup-deploy.overrides.sh#L29
- https://github.com/bcgov/family-law-act-app/blob/master/openshift/templates/backup/backup-deploy.overrides.sh#L34

Reply:
change to only allow ${SMTP_EMAIL_HOST}